tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -485,6 +512,32 @@ stages:
               ls -la ${{ variables.memoryUsageTestOutputFolder }};
               node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/memoryUsageTestHandler.js --dir ${{ variables.memoryUsageTestOutputFolder }};
 
+        - task: Bash@3
+          displayName: Send Excecution Time Perf Benchmark Measurments to Azure App Insights
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing memory usage performance benchmark output to Azure App Insights..."
```

Comment:
This echo and the one in line 534 should be swapped

Reply:
Fixed in recent commit # 84045